Mathlib/Data/ENNReal/Real.lean

```diff
@@ -274,6 +274,7 @@ theorem lt_ofReal_iff_toReal_lt {a : ℝ≥0∞} {b : ℝ} (ha : a ≠ ∞) :
 theorem toReal_lt_of_lt_ofReal {b : ℝ} (h : a < ENNReal.ofReal b) : ENNReal.toReal a < b :=
   (lt_ofReal_iff_toReal_lt h.ne_top).1 h
 
+@[simp]
```

Comment:
This one looks a lot like the `map_mul` lemmas that normally do not have a `simp` tag: why is this one different?

Reply:
In `map_mul f a b : f (a * b) = f a * f b`, the free variable `f` appears more often on the RHS than on the LHS. In `ofReal_mul p q : ofReal (p * q) = ofReal p * ofReal q`, this doesn't happen, because `ofReal` isn't a free variable